.github/pr-labeler.yml

```diff
@@ -0,0 +1,4 @@
+feature: ['feature/*', 'feat/*']
+fix: fix/*
+chore: ['chore/*', 'deps/*']
```

Comment:
hm. we already have the "dependencies" label. shall we use dependencies for deps/*?